### DISCUSSION TITLE
Download failed

### DISCUSSION THREAD

**Discussion by RC-FIXER** [aclap-dev/video-downloadhelper#603] [Help and Q&A]
### Your question

{
  "version": "9.0.1.13",
  "target": "mozilla",
  "channel": "stable",
  "page_url": "undef",
  "error": {
    "error": "coapp_failure",
    "details": "SideDownload error: : [tls @ 000002873d71c0c0] IO error: End of file\r\n[https @ 000002873d71a500] Stream ends prematurely at 1601249, should be 903871926\r\n[mov,mp4,m4a,3gp,3g2,mj2 @ 000002873bf589c0] error reading header\r\n[in#0 @ 000002873bf58800] Error opening input: End of file\r\nError opening input file https://serverporno10.xyz/media/videos/h264/59838_720p.mp4.\r\nError opening input files: End of file\r\n",
    "id": "coapp_failure_20358d0b-a64a-479f-909a-512903582fae",
    "downloadable_id": "downloadable_7484834618770147"
  }
}

### Addon & CoApp version and details

version: 9.0.1.13
target: mozilla
channel: stable
lang: pt-BR
coapp: {"found":true,"path":"C:\\Program Files\\DownloadHelper CoApp\\vdhcoapp.exe","version":"2.0.19","new_version":false}
license: {"unset":true}
platform: x86-64 win
UA: Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:126.0) Gecko/20100101 Firefox/126.0
{
    "http_media_download_strategy": "coapp",
    "used_sidebar_button": true,
    "used_popup_button": true,
    "download_directory": "C:\\Users\\User\\Downloads",
    "theme": "system",
    "view_options": {
        "all_tabs": false,
        "low_quality": false,
        "sort_by_status": true,
        "sort_reverse": false,
        "show_button_clean": true,
        "show_button_clean_all": false,
        "show_button_convert_local": false,
        "hide_downloaded": false
    },
    "use_wide_ui": true,
    "use_sidebar": false,
    "open_count_store": 28
}

**Comment by paulrouget** [comment]
Note to self: ff-failure:  Stream ends prematurely

**Comment by GrampaWildWilly** [comment]
Right.  VDH cut it off about 4 minutes before the end.  But I ran ffmpeg directly on it & it read the full 50:41 of this.  This is the ffmpeg that the CoApp uses.  So this is quite mysterious.
